layer-1-contracts/README.md

```diff
@@ -0,0 +1,19 @@
+# Layer 1 Contracts
+
+## Design Goals
+We want our ORU contracts to be built in compliance with the following goals:
+- A **pluggable state machine** which:
+    - allows separation of concerns and parallelizable work
+    - is fully stateless, i.e. all state transitions can be expressed in the form `bytes32 _inputHash, bytes _witness => bytes32 _outputhash` and are only dependent on the immediately preceding pre-state
+- A cryptoeconomic **State Oracle** which:
+    - Provides a cryptoeconomic mechanism to propose, assert, challenge, and finalize optimistic states of the L2 machine
+        - Does NOT require that the entire execution trace is provable, just the execution output
+            - i.e. an honest party may not necessarily ever post a hash of the full trace
```

Comment:
This callout is one of the assumptions allowing #3 [here](https://statechannels.notion.site/Various-bisection-games-8d5ed428f7114b52a1c6bbeb08fe96d5)

Reply:
FYI `#3` links to pull request #3 in this repo, which is misleading. I think you mean "Option 3" in the linked notion doc, right?